server/matchmaker/search.py

```diff
@@ -59,7 +61,7 @@ def _is_ladder_newbie(player: Player) -> bool:
         return player.game_count[RatingType.LADDER_1V1] <= config.NEWBIE_MIN_GAMES
 
     def is_ladder1v1_search(self) -> bool:
-        return self.rating_type is RatingType.LADDER_1V1
+        return self.rating_type == RatingType.LADDER_1V1
```

Comment:
I thought enums are supposed to be "compared by identity"
https://docs.python.org/3/library/enum.html#comparisons

Reply:
Oh except I see below that they aren't enums anymore but strings?

Reply:
Yea it can be either, since we're using this `RatingType` enum for the parts that are hardcoded, but of course data from the DB is just going to be strings. The other alternative is ditching the enum altogether and just making it a class with some string constants.